const errorCatch = (err) => {
    console.log('Error', err)
}

module.exports = { errorCatch }